src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -1367,7 +1369,7 @@ override public string GetName(int i)
         }
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml' path='docs/members[@name="SqlDataReader"]/GetProviderSpecificFieldType/*' />
-#if NET8_0_OR_GREATER
+#if !NETFRAMEWORK && !NET6_0
```

Comment:
Hmm.. I think this one should probably stay as `NET8_0_OR_GREATER`. I probably am making a mess of these suggestions here, but my thought is that if it's differentiating between versions of netcore then it's fine to leave as `NETx_OR_GREATER`, but if it's to differentiate between netfx and netcore, then I think it's better as `NETFRAMEWORK`/`!NETFRAMEWORK`

But a better question for this one might be - why was it NET**8**_0_OR_GREATER in the first place?

Reply:
That makes sense, thanks - I'll use `!NETFRAMEWORK && NET8_OR_GREATER`, so it doesn't get removed by accident when we remove .NET 6.0 support.

The annotation's different because this overrides DbDataReader.GetProviderSpecificFieldType. Trimming annotations need to match between base and derived classes, and the annotation on the base class method changed between 6.0 and 8.0.